src/components/pages/adminHome/adminSidebar.js

```diff
@@ -43,23 +42,23 @@ function AdminSidebar() {
         </Menu.Item>
 
         <Menu.Item key="2" icon={<HomeOutlined />}>
-          <a href="#">Dashboard</a>
+          <a href="/admin">Dashboard</a>
         </Menu.Item>
 
         <Menu.Item key="3" icon={<BellOutlined />}>
-          <a href="#">Notifications</a>
+          <a href="!#">Notifications</a>
         </Menu.Item>
 
         <Menu.Item key="4" icon={<CalculatorOutlined />}>
-          <a href="#">Analytics</a>
+          <a href="!#">Analytics</a>
         </Menu.Item>
 
         <Menu.Item key="5" icon={<DesktopOutlined />}>
-          <a href="#">Courses</a>
+          <a href="!#">Courses</a>
         </Menu.Item>
 
         <Menu.Item key="6" icon={<PlusOutlined />}>
-          <a href="#">Add Courses</a>
+          <a href="!#">Add Courses</a>
```

Comment:
What is the purpose of these !s?

Reply:
In short, these are preventing the browser from moving your view to the top of the page when you click these currently empty links. 🙂